src/main/java/de/muenchen/mobidam/s3/S3RouteBuilder.java

```diff
@@ -55,6 +55,8 @@ public void configure() {
                             log.error(
                                     String.format("Archive clean up (%s): %s", exchange.getProperty(Constants.ARCHIVE_ENTITY, MobidamArchive.class).toString(),
                                             res.getError()));
+                        } else {
```

Comment:
@sfi2022 es geht hier eher darum, die Exception, die sich in den internalProperties befindet, auszugeben.

Reply:
@martind260 Ich denke das Problem liegt in dieser Zeile 
log.error("Error occurred in route", exception); 
Die Erwartung war, das die Exception mit ausgegeben wird. Das ist wohl nicht der Fall. Ich ändere das. 
Die ursprüngliche Änderung mache ich wieder Rückgängig.